### DISCUSSION TITLE
UX is poor with everything needing "answer", not everything is an answer.

### DISCUSSION THREAD

**Discussion by ElijahLynn** [community/community#4309] [Discussions]
The UX is confusing in Discussions because it seems everything needs to be an "Answer", which isn't applicable to all conversations/discussions. 

And when I just went to create this discussion I see now in the dropdown that there is an "Answers enabled": 
![image](https://user-images.githubusercontent.com/1504756/123337386-f8253580-d4fb-11eb-8461-2ee04bf79a01.png)

At any rate, just want to say this is all confusing when I am new to discussions and just trying to discuss something, not answer something.

**Comment by Zerotask** [comment]
Totally agree with you. Changing the button to `Reply` would be better to understand and more consistent with the alternative interaction of reacting (set an emoji)

**Comment by tonygermano** [comment]
Duplicate of https://github.com/github/feedback/discussions/3798 regarding Discussions areas where answers are enabled.
